README.md

```diff
@@ -33,6 +33,20 @@ discussions, and more.
 ### [Cardigan](https://cardigan.wellcomecollection.org)
 - Wellcome Collection's design system. [`code`](./cardigan).
 
+## Developing
+
+This project uses [Yarn](https://yarnpkg.com/lang/en/). You'll need to install it before proceeding.
+
+To start a hot reload of a service run:
+
+```
+# Install dependencies
```

Comment:
As we use workspaces, dependencies are managed from the root directory.
I also haven't seen the run command anywhere?

Suggestion
---

## Local development

We use [Yarn](https://yarnpkg.com/lang/en/) to manage out external dependencies.

We then use [Yarn Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) to manage our [local, common dependencies](https://github.com/wellcometrust/wellcomecollection.org/tree/master/common).

To run a project, from the root directory:
```bash
yarn install
# yarn run {appName = catalogue|content}
# e.g.
yarn run catalogue
```

---


I might add a piece about how the webapps work with middleware etc.

Reply:
Much nicer - changed as per suggestion. I think you can offer code changes directly on PRs too.